cmake/modules/FindNVSHMEM.cmake

```diff
@@ -67,16 +81,3 @@ mark_as_advanced(FORCE NVSHMEM_INCLUDE_DIRS)
 # Set the libraries
 set(NVSHMEM_LIBRARIES NVSHMEM::NVSHMEM)
 mark_as_advanced(FORCE NVSHMEM_LIBRARY)
-
-if (NVSHMEM_FOUND)
-  set_property(TARGET cuda::toolkit APPEND PROPERTY
-    INTERFACE_LINK_LIBRARIES ${CUDA_cudadevrt_LIBRARY})
-  # Build static libraries to get around a bug in NVSHMEM
-  set(BUILD_SHARED_LIBS OFF)
```

Comment:
If I remember correctly, we forced building a static library because position-independent code didn't work for CUDA device functions, which caused problems with NVSHMEM's device functions.

Reply:
Addressed in commit 18446c2.